lua/lib/renderer.lua

```diff
@@ -48,6 +48,21 @@ if icon_state.show_file_icon then
 
 end
 
+local get_symlink_icon = function() return icon_state.icons.symlink end
+if icon_state.show_file_icon then
+  get_symlink_icon = function()
+    return #icon_state.icons.symlink > 0 and icon_state.icons.symlink.." " or ""
+  end
+end
+
+local get_special_icon = function() return icon_state.icons.default end
+if icon_state.show_file_icon then
+  get_special_icon = function()
+    -- Special files don't have an icon
+    return " "
```

Comment:
so they shouldn't return a space ? Please reimplement the previous logic because some people want to display a default icon for each file, even special ones.

Reply:
Fixed